modules/admin-ui/src/main/java/org/opencastproject/adminui/endpoint/SeriesEndpoint.java

```diff
@@ -1039,4 +1087,16 @@ private boolean hasProcessingEvents(String seriesId) {
     return Opt.some(result.getItems()[0].getSource());
   }
 
+  @GET
+  @Path("options.json")
```

Comment:
I was a bit confused by this name and did not expect this to return the service configuration. Maybe call this `/configuration.json`?